airflow/www/static/js/connection_form.js

```diff
@@ -234,11 +234,18 @@ $(document).ready(() => {
   /**
    * Displays the Flask style alert on UI via JS
    *
-   * @param {boolean} status - true for success, false for error
+   * @param {boolean} status - true for success, false for error, null for warning
```

Comment:
This gets confusing now, let's make this a string instead: 'success', 'warning', 'error'

Reply:
@bbovenzi Thanks for clarification! I made those changes if you would like to review them